core/domain/suggestion_services.py

```diff
@@ -348,3 +350,107 @@ def reject_suggestion(suggestion, reviewer_id, review_message):
     feedback_services.create_message(
         get_thread_id_from_suggestion_id(suggestion.suggestion_id), reviewer_id,
         feedback_models.STATUS_CHOICES_IGNORED, None, review_message)
+
+
+def get_user_contribution_scoring_from_model(userContributionScoringModel):
+    """Returns the UserContributionScoring domain object corresponding to the
+    UserContributionScoringModel
+
+    Args:
+        userContributionScoringModel: UserContributionScoringModel. The model
+            instance.
+
+    Returns:
+        UserContributionScoring. The corresponding domain object.
+    """
+    return user_domain.UserContributionScoring(
+        userContributionScoringModel.user_id,
+        userContributionScoringModel.score_category,
+        userContributionScoringModel.score)
+
+
+def get_all_scores_of_user(user_id):
```

Comment:
When would this function be used?

Reply:
I was thinking if we ever need to query all categories the user can review or even just display all the scores of the user (though we did decide against this for the moment, in the future if needed we could use this function).